spec/helpers/upvotes_helper_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the UpvotesHelper. For example:
+#
+# describe UpvotesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```

Comment:
I abhor comment blocks like this. If you aren't going to use it delete it.

Oh and also either add tests or delete these files. ;)